charts/tempo-distributed/templates/distributor/persistentvolumeclaim-compactor.yaml

```diff
@@ -0,0 +1,17 @@
+{{- if .Values.compactor.persistence.enabled }}
```

Comment:
compactors don't use the local disk.  this should not be necessary.

Reply:
Okay. Just for double checking, I know Loki uses it to reduce start up time. Is this something Tempo can do? or will do in the future?

Reply:
I also see this is in the wrong folder 😅

Reply:
at the moment we have no plans to use a disk for the compactor.  i wonder if they're using it to persist tokens?  

even if we did decide to add disks for token persistence we'd have to change the compactor deployment to a statefulset.  so it would require additional work.

Reply:
I didn't check what they store, but you can keep the deployment as a deployment 😄 At least Loki-compactor is running as a deployment and run fines.